lib/tuya.js

```diff
@@ -1460,6 +1460,22 @@ const valueConverter = {
     },
     inverse: {to: (v) => !v, from: (v) => !v},
     onOffNotStrict: {from: (v) => v ? 'ON' : 'OFF', to: (v) => v === 'ON'},
+    errorBatteryLow: {
+        from: (v) => {
+            if (v === 0) return {'battery_low': false};
+            if (v === 1) return {'battery_low': true};
+            return {'error': v};
+        },
+    },
+    idleHeat: valueConverterBasic.lookup({'idle': 1, 'heat': 0}),
+    autoHeatOff: valueConverterBasic.lookup({'auto': 0, 'heat': 1, 'off': 2}),
```

Comment:
These 2 seem to be very specific to the thermostat, can you move them to the device?

Reply:
done